src/livestreamer/plugins/tv3cat.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+import re
+import logging
+
+from livestreamer.plugin import Plugin
+from livestreamer.plugin.api import http
+from livestreamer.stream import HLSStream
+from livestreamer.plugin.api import validate
+
+
+log = logging.getLogger(__name__)
```

Comment:
I guess you left this by accident? For future reference, you can use `self.logger.[error|warning|info|debug]` in the plugin.